quiz.js

```diff
@@ -47,24 +47,29 @@ var characterField = document.getElementById("character");
 characterField.addEventListener("keypress", handleKeyPress);
 
 
+
 function handleKeyPress(e){	 // when you press enter instead of click, had to add the function name to both the input tags
- var key=e.keyCode;
-  if (key==13){
-  	movement();
+  var key=e.keyCode;
+  	if (key==13){
+  	  movement();
   }
 }
 
+var treeData = {}; // needs to be globally defined not related to the function movement
 
 function movement() { //middle function to make objects to be passed below
```

Comment:
All the indentation above this point in the code is done with spaces, and from here down it switches to tabs.

Not sure if it's just Github being weird about whitespace, but if not, make sure you use the same indentation style throughout your code.